src/shared/helpers.js

```diff
@@ -0,0 +1,7 @@
+export const formatCid = (cid) => {
```

Comment:
I'd rename the file from `helpers.js` to `formatCID.js`

Reply:
This is a helper method, maybe we'll have other helpers methods? We'll create for each method a new file?